src/net/rdma.cc

```diff
@@ -199,7 +208,7 @@ future<> RDMAConnection::completeHandshake(uint32_t remoteQP) {
                                     IBV_QP_RQ_PSN | IBV_QP_MAX_DEST_RD_ATOMIC |
                                     IBV_QP_MIN_RNR_TIMER | IBV_QP_DEST_QPN)) {
             K2ERROR("failed to transition RC QP into RTR: " << strerror(err));
-            throw RDMAConnectionError();
+            return err;
```

Comment:
While you're here, can you also log the strerror(errno)? Not sure if the api has them the same or not

Reply:
Justin checked - API returns errno on error